src/main/webapp/fellowship/index.html

```diff
@@ -0,0 +1,108 @@
+<!-- Optional JS -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/14.0.3/nouislider.min.js"></script>
+<!-- Argon JS -->
+<script src="../assets/js/argon.js?v=1.0.1"></script>
+<script src="../assets/js/global-custom-js.js"></script>
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
+<!-- Script for static navbar and footer -->
+<script src="../assets/js/navigation.js"></script>
+<script src="../assets/js/functions.js"></script>
+
+<script>
+    $(function () {
+        loadNavAndFooter('/assets/content/static');  //relative path to content directory
+    });
+</script>
+<script type="text/html" id="certificate-template">
+    <div class="h-100vh justify-content-center align-items-center row"
+         style="background: url('images/certificate_background.svg') right no-repeat"
+    >
+        <div class="shadow bg-white m-5">
+            <div class="container">
+                <div class="row p-md-5 p-3 position-relative">
+                    <img class="position-absolute h-100 d-none d-md-block certificate-bg-image"
+                         src="images/certificate_background.svg"
+                    >
+                    <div class="col-md-8">
+                        <h2 class="text-dark font-weight-900">Certificate of Fellowship</h2>
+                        <div class="mt-md-4">
+                            <p class="small text-dark">This certificate is awarded to</p>
+                            <h3 class="text-muted">{{name}}</h3>
+                            <p class="text-dark">
+                                The  Fellowship program is an initiative to recognise individuals who have actively
+                                contributed to the delivery of flagship programs of the Sustainable Education Foundation
+                            </p>
+                            <p class="font-italic text-muted small">
+                                DIGITALLY SIGNED FOR AND ON BEHALF OF THE SUSTAINABLE EDUCATION FOUNDATION
+                            </p>
+                        </div>
+                        <div class="mt-5">
+                            <p class="small">
+                                Date Issued: <span class="text-dark">{{createdAt}}</span><br>
+                                Certificate Id: <span class="text-dark">{{id}}</span>
+                            </p>
+                            <img src="../assets/img/brand/logo.png" class="img-fluid w-25">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</script>
+<script>
+    $(function () {
+        let searchParams =  new URLSearchParams(window.location.search)
+        if(searchParams.has('id')){
+            let id = atob(searchParams.get('id'));
+            $.ajax({
+                type: 'get',
+                url: 'http://localhost:8080/core/fellowship/certificates/'+id,
```

Comment:
This URL should be changed

Reply:
Sorry, My mistake. Let me fix it.

Reply:
Resolved with commit https://github.com/sef-global/sef-site/pull/748/commits/f2d6f67a13c6bbc62464767011b16e79e033fa6b